ORBIT/manager.py

```diff
@@ -189,14 +195,19 @@ def compile_input_dict(cls, phases):
             config = cls.merge_dicts(config, d.expected_config)
             config = cls.remove_keys(config, d.output_config)
 
-        config["commissioning"] = "float (optional, default: 0.01)"
-        config["decommissioning"] = "float (optional, default: 0.15)"
-
-        config["ncf"] = "float (optional, default: 0.4)"
-        config["offtake_price"] = "$/MWh (optional, default: 80)"
-        config["project_lifetime"] = "yrs (optional, default: 25)"
-        config["discount_rate"] = "yearly (optional, default: .025)"
-        config["opex_rate"] = "$/kW/year (optional, default: 150)"
+        config["project_parameters"] = {
+            "turbine_capex": "$/kW (optional, default: 0.0)",
+            "ncf": "float (optional, default: 0.4)",
+            "offtake_price": "$/MWh (optional, default: 80)",
+            "project_lifetime": "yrs (optional, default: 25)",
+            "discount_rate": "yearly (optional, default: .025)",
+            "opex_rate": "$/kW/year (optional, default: 150)",
+            "construction_insurance": "$/kW (optional, default: 44)",
+            "construction_financing": "$/kW (optional, default: 183)",
+            "contingency": "$/kW (optional, default: 316)",
+            "commissioning": "$/kW (optional, default: 44)",
+            "decommissioning": "$/kW (optional, default: 58)",
+        }
```

Comment:
@Matt-Shields This is the key change. Project related inputs are now expected to be in the "project_parameters" dictionary. The NPV related inputs are also in here too.